composer.json

```diff
@@ -14,7 +14,8 @@
   "require": {
     "php": "^7.1",
     "illuminate/support": "5.0 - 5.8 | ^6.0",
-    "sentry/sdk": "^2.0"
+    "sentry/sdk": "^2.0",
+    "sentry/sentry": "dev-develop@dev"
```

Comment:
I would advise against doing this in the code. For `sentry-symfony`, I was thinking about adding a dedicated job in the CI that would programmatically do the same. Maybe you can do it too?

Reply:
This is why the PR is a draft 😉 This is not meant to go into the actual PR but for now the only way to pull in the correct code I believe.

Reply:
Added the develop branch to the test matrix too.